dts/bindings/rtc/microcrystal,rv-8263-c8.yaml

```diff
@@ -0,0 +1,43 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) 2024 Daniel Kampert
+# Author: Daniel Kampert <DanielKampert@Kampis-Elektroecke.de>
+
+description: Micro Crystal RV-8263-C8 RTC
+
+compatible: "microcrystal,rv-8263-c8"
+
+include:
+  - name: rtc-device.yaml
+  - name: i2c-device.yaml
+
+properties:
+    int-gpios:
+      type: phandle-array
+
+    fast_mode:
+      type: boolean
+      description: |
+        Enable fast offset mode (Offset is made every 4 minutes).
+        Default is 0 and corrosponds to the reset value of the register field.
```

Comment:
missed removing this from the description when default was removed 

booleans are always false by default

Reply:
But isn't the description for the default value right when default is false?

Reply:
It's redundant to clarify that a boolean is false by default, please remove it